src/init.c

```diff
@@ -37,6 +39,24 @@ extern void _PG_fini(void);
 void
 _PG_init(void)
 {
+	if (!process_shared_preload_libraries_in_progress)
+	{
+	
+		char	   *force_load = GetConfigOptionByName("timescaledb.force_load", NULL, true);
+		if (force_load == NULL || strlen(force_load) != 2 || strncmp(force_load, "on", 2) != 0) {
+			char	   *config_file = GetConfigOptionByName("config_file", NULL, false);
+			elog(ERROR,
+"The timescaledb library is not preloaded\n\n"
+"We strongly recommend loading the timescaledb library via shared_preload_libraries.\n\n"
+"This should be done by editing the config file at: %1$s\n"
+"and adding 'timescaledb' to the list in the shared_preload_libraries config.\n"
+"An easy way to do this, if not preloading other libraries, is with the bash command:\n"
+"	echo \"shared_preload_libraries = 'timescaledb'\" >> %1$s \n\n"
```

Comment:
I guess this is fine, but might not work if this line is already uncommented, albeit set to the empty string.